### PR TITLE
Bug 1383268 - Add attachments to new-bug

### PR DESCRIPTION
**Pull request by SebastinSanty**

### CODE REVIEW

File: extensions/BugModal/web/new_bug.css [hunk 1]
```diff
@@ -44,3 +48,7 @@
 input, label {
     display: block; !important
```
Comment (dylanwh) [inline extensions/BugModal/web/new_bug.css]: just noticed this, the ; should be after the !important.

File: new_bug.cgi [hunk 2]
```diff
@@ -45,6 +45,8 @@ my $user = Bugzilla->login(LOGIN_REQUIRED);
 my $cgi      = Bugzilla->cgi;
 my $template = Bugzilla->template;
 my $vars     = {};
+my $dbh = Bugzilla->dbh;
```
Comment (dylanwh) [inline new_bug.cgi]: align with other ='s (perltidy will do this usually)

File: new_bug.cgi [hunk 3]
```diff
@@ -65,6 +67,34 @@ if (lc($cgi->request_method) eq 'post') {
...
+         Bugzilla->error_mode(ERROR_MODE_DIE);
+         my $timestamp = $dbh->selectrow_array(
+             'SELECT creation_ts FROM bugs WHERE bug_id = ?', undef, $new_bug->bug_id);
+         eval {
+             $attachment = Bugzilla::Attachment->create(
+                 {bug           => $new_bug,
+                  creation_ts   => $timestamp,
+                  data          => $data_fh,
+                  description   => scalar $cgi->param('description'),
+                  filename      => $data_fh,
```
Comment (dylanwh) [inline new_bug.cgi]: the filename can't be $data_fh -- how do we get the file name in post_bug.cgi?

Comment (SebastinSanty) [inline new_bug.cgi]: In post_bug.cgi, it is `$cgi->param('attach_text') || $data_fh`

**Pull request merged**